CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+## Development
+
+First of all, thank you for your interested in this project. Please read the documentation here to being contributing. 
+
+### Creating an Action
+
+- :file_folder: In a new branch create an new folder in a root folder of this repository and name it to your Action name.
+- :octocat: Follow the instructions [Creating an Action with the Toolkit](https://github.com/actions/toolkit#creating-an-action-with-the-toolkit) to create an Action.
+- :construction: Start developing and open a pull request.
+
+Read more about GitHub Actions [here](https://help.github.com/en/articles/getting-started-with-github-actions).
+
+### Recommendations on versioning your action
+
+Since we keep multiple actions in one repository, all the actions are released in one batch (we do not support semantic versioning). 
+
+**JavaScript / TypeScript Action**
+
+When your pull request is merged:
```

Comment:
After or while it is being merged?
Are these instructions to prepare a release?
Why does a PR contributor have to do that? Isn't that part of a release process?

Reply:
> After or while it is being merged?

When your pull request was merged. 

> Are these instructions to prepare a release?

Basically yes 👍  

> Why does a PR contributor have to do that? Isn't that part of a release process?

Should, the project maintainers could do that. Where should we put a release description then? 🤔

Reply:
Section "How to release new versions" and then maybe a "Steps to prepare the code" and "How to do the actual release" or similar 👍

Reply:
I changed the section title to `How to release new versions`. I think splitting the others will be a bit confusing. Right now we have two path to release the action based on its type: node Action or Docker Action.  WDYT?